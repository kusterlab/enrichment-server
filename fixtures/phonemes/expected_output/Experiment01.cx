[{"numberVerification":[{"longNumber":281474976710655}]},{"metaData":[{"name":"nodeAttributes","elementCount":8,"version":"1.0"},{"name":"cyGroups","version":"1.0"},{"name":"cyTableColumn","elementCount":15,"version":"1.0"},{"name":"edges","elementCount":10,"idCounter":212,"version":"1.0"},{"name":"cyVisualProperties","elementCount":3,"version":"1.0"},{"name":"cyHiddenAttributes","elementCount":1,"version":"1.0"},{"name":"nodes","elementCount":8,"idCounter":176,"version":"1.0"},{"name":"cyNetworkRelations","version":"1.0"},{"name":"networkAttributes","elementCount":1,"version":"1.0"},{"name":"cartesianLayout","elementCount":8,"version":"1.0"},{"name":"edgeAttributes","elementCount":60,"version":"1.0"},{"name":"tableVisualProperties","version":"1.0"},{"name":"cySubNetworks","version":"1.0"}]},{"cyTableColumn":[{"applies_to":"node_table","n":"shared name"},{"applies_to":"node_table","n":"name"},{"applies_to":"node_table","n":"id"},{"applies_to":"edge_table","n":"shared name"},{"applies_to":"edge_table","n":"shared interaction"},{"applies_to":"edge_table","n":"name"},{"applies_to":"edge_table","n":"interaction"},{"applies_to":"edge_table","n":"source"},{"applies_to":"edge_table","n":"target"},{"applies_to":"edge_table","n":"sign","d":"integer"},{"applies_to":"edge_table","n":"weight","d":"integer"},{"applies_to":"edge_table","n":"data.key.column","d":"integer"},{"applies_to":"network_table","n":"shared name"},{"applies_to":"network_table","n":"name"},{"applies_to":"network_table","n":"__Annotations","d":"list_of_string"}]},{"networkAttributes":[{"n":"name","v":"Experiment01"}]},{"nodes":[{"@id":176,"n":"MAPKAPK5"},{"@id":173,"n":"MAPKAPK2"},{"@id":170,"n":"MAPK14"},{"@id":167,"n":"MAPK1"},{"@id":164,"n":"MAP2K1"},{"@id":161,"n":"EGFR"},{"@id":158,"n":"MAP3K8"},{"@id":151,"n":"AKT2"}]},{"edges":[{"@id":212,"s":170,"t":176,"i":"interacts with"},{"@id":209,"s":170,"t":173,"i":"interacts with"},{"@id":206,"s":170,"t":151,"i":"interacts with"},{"@id":203,"s":167,"t":176,"i":"interacts with"},{"@id":200,"s":167,"t":173,"i":"interacts with"},{"@id":194,"s":164,"t":170,"i":"interacts with"},{"@id":191,"s":164,"t":167,"i":"interacts with"},{"@id":188,"s":161,"t":164,"i":"interacts with"},{"@id":197,"s":158,"t":167,"i":"interacts with"},{"@id":179,"s":151,"t":158,"i":"interacts with"}]},{"nodeAttributes":[{"po":176,"n":"id","v":"MAPKAPK5"},{"po":173,"n":"id","v":"MAPKAPK2"},{"po":170,"n":"id","v":"MAPK14"},{"po":167,"n":"id","v":"MAPK1"},{"po":164,"n":"id","v":"MAP2K1"},{"po":161,"n":"id","v":"EGFR"},{"po":158,"n":"id","v":"MAP3K8"},{"po":151,"n":"id","v":"AKT2"}]},{"edgeAttributes":[{"po":212,"n":"sign","v":"1","d":"integer"},{"po":212,"n":"name","v":"MAPK14 (interacts with) MAPKAPK5"},{"po":212,"n":"weight","v":"100","d":"integer"},{"po":212,"n":"data.key.column","v":"212","d":"integer"},{"po":212,"n":"source","v":"MAPK14"},{"po":212,"n":"target","v":"MAPKAPK5"},{"po":209,"n":"sign","v":"1","d":"integer"},{"po":209,"n":"name","v":"MAPK14 (interacts with) MAPKAPK2"},{"po":209,"n":"weight","v":"100","d":"integer"},{"po":209,"n":"data.key.column","v":"209","d":"integer"},{"po":209,"n":"source","v":"MAPK14"},{"po":209,"n":"target","v":"MAPKAPK2"},{"po":206,"n":"sign","v":"1","d":"integer"},{"po":206,"n":"name","v":"MAPK14 (interacts with) AKT2"},{"po":206,"n":"weight","v":"100","d":"integer"},{"po":206,"n":"data.key.column","v":"206","d":"integer"},{"po":206,"n":"source","v":"MAPK14"},{"po":206,"n":"target","v":"AKT2"},{"po":203,"n":"sign","v":"1","d":"integer"},{"po":203,"n":"name","v":"MAPK1 (interacts with) MAPKAPK5"},{"po":203,"n":"weight","v":"100","d":"integer"},{"po":203,"n":"data.key.column","v":"203","d":"integer"},{"po":203,"n":"source","v":"MAPK1"},{"po":203,"n":"target","v":"MAPKAPK5"},{"po":200,"n":"sign","v":"1","d":"integer"},{"po":200,"n":"name","v":"MAPK1 (interacts with) MAPKAPK2"},{"po":200,"n":"weight","v":"100","d":"integer"},{"po":200,"n":"data.key.column","v":"200","d":"integer"},{"po":200,"n":"source","v":"MAPK1"},{"po":200,"n":"target","v":"MAPKAPK2"},{"po":194,"n":"sign","v":"1","d":"integer"},{"po":194,"n":"name","v":"MAP2K1 (interacts with) MAPK14"},{"po":194,"n":"weight","v":"50","d":"integer"},{"po":194,"n":"data.key.column","v":"194","d":"integer"},{"po":194,"n":"source","v":"MAP2K1"},{"po":194,"n":"target","v":"MAPK14"},{"po":191,"n":"sign","v":"1","d":"integer"},{"po":191,"n":"name","v":"MAP2K1 (interacts with) MAPK1"},{"po":191,"n":"weight","v":"50","d":"integer"},{"po":191,"n":"data.key.column","v":"191","d":"integer"},{"po":191,"n":"source","v":"MAP2K1"},{"po":191,"n":"target","v":"MAPK1"},{"po":188,"n":"sign","v":"1","d":"integer"},{"po":188,"n":"name","v":"EGFR (interacts with) MAP2K1"},{"po":188,"n":"weight","v":"50","d":"integer"},{"po":188,"n":"data.key.column","v":"188","d":"integer"},{"po":188,"n":"source","v":"EGFR"},{"po":188,"n":"target","v":"MAP2K1"},{"po":197,"n":"sign","v":"1","d":"integer"},{"po":197,"n":"name","v":"MAP3K8 (interacts with) MAPK1"},{"po":197,"n":"weight","v":"50","d":"integer"},{"po":197,"n":"data.key.column","v":"197","d":"integer"},{"po":197,"n":"source","v":"MAP3K8"},{"po":197,"n":"target","v":"MAPK1"},{"po":179,"n":"sign","v":"1","d":"integer"},{"po":179,"n":"name","v":"AKT2 (interacts with) MAP3K8"},{"po":179,"n":"weight","v":"50","d":"integer"},{"po":179,"n":"data.key.column","v":"179","d":"integer"},{"po":179,"n":"source","v":"AKT2"},{"po":179,"n":"target","v":"MAP3K8"}]},{"cartesianLayout":[{"node":161,"x":7.03125,"y":-213.125},{"node":164,"x":7.03125,"y":-148.125},{"node":167,"x":7.03125,"y":156.875},{"node":170,"x":-11.71875,"y":-68.125},{"node":173,"x":-45.46875,"y":251.875},{"node":176,"x":59.53125,"y":251.875},{"node":151,"x":-11.71875,"y":11.875},{"node":158,"x":-11.71875,"y":76.875}]},{"cyVisualProperties":[{"properties_of":"network","properties":{"NETWORK_ANNOTATION_SELECTION":"false","NETWORK_BACKGROUND_PAINT":"#FFFFFF","NETWORK_CENTER_X_LOCATION":"7.03125","NETWORK_CENTER_Y_LOCATION":"19.375","NETWORK_CENTER_Z_LOCATION":"0.0","NETWORK_DEPTH":"0.0","NETWORK_EDGE_SELECTION":"true","NETWORK_FORCE_HIGH_DETAIL":"false","NETWORK_HEIGHT":"366.0","NETWORK_NODE_LABEL_SELECTION":"false","NETWORK_NODE_SELECTION":"true","NETWORK_SCALE_FACTOR":"0.71736","NETWORK_SIZE":"550.0","NETWORK_WIDTH":"816.0"}},{"properties_of":"nodes:default","properties":{"COMPOUND_NODE_PADDING":"10.0","COMPOUND_NODE_SHAPE":"ROUND_RECTANGLE","NODE_BORDER_PAINT":"#CCCCCC","NODE_BORDER_STROKE":"SOLID","NODE_BORDER_TRANSPARENCY":"255","NODE_BORDER_WIDTH":"0.0","NODE_CUSTOMGRAPHICS_1":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_2":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_3":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_4":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_5":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_6":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_7":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_8":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_9":"org.cytoscape.cg.model.NullCustomGraphics,0,[ Remove Graphics ],","NODE_CUSTOMGRAPHICS_POSITION_1":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_2":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_3":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_4":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_5":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_6":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_7":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_8":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_POSITION_9":"C,C,c,0.00,0.00","NODE_CUSTOMGRAPHICS_SIZE_1":"50.0","NODE_CUSTOMGRAPHICS_SIZE_2":"50.0","NODE_CUSTOMGRAPHICS_SIZE_3":"50.0","NODE_CUSTOMGRAPHICS_SIZE_4":"50.0","NODE_CUSTOMGRAPHICS_SIZE_5":"50.0","NODE_CUSTOMGRAPHICS_SIZE_6":"50.0","NODE_CUSTOMGRAPHICS_SIZE_7":"50.0","NODE_CUSTOMGRAPHICS_SIZE_8":"50.0","NODE_CUSTOMGRAPHICS_SIZE_9":"50.0","NODE_CUSTOMPAINT_1":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_1, name=Node Custom Paint 1)","NODE_CUSTOMPAINT_2":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_2, name=Node Custom Paint 2)","NODE_CUSTOMPAINT_3":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_3, name=Node Custom Paint 3)","NODE_CUSTOMPAINT_4":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_4, name=Node Custom Paint 4)","NODE_CUSTOMPAINT_5":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_5, name=Node Custom Paint 5)","NODE_CUSTOMPAINT_6":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_6, name=Node Custom Paint 6)","NODE_CUSTOMPAINT_7":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_7, name=Node Custom Paint 7)","NODE_CUSTOMPAINT_8":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_8, name=Node Custom Paint 8)","NODE_CUSTOMPAINT_9":"DefaultVisualizableVisualProperty(id=NODE_CUSTOMPAINT_9, name=Node Custom Paint 9)","NODE_DEPTH":"0.0","NODE_FILL_COLOR":"#89D0F5","NODE_HEIGHT":"35.0","NODE_LABEL_COLOR":"#000000","NODE_LABEL_FONT_FACE":"SansSerif.plain,plain,12","NODE_LABEL_FONT_SIZE":"12","NODE_LABEL_POSITION":"C,C,c,0.00,0.00","NODE_LABEL_ROTATION":"0.0","NODE_LABEL_TRANSPARENCY":"255","NODE_LABEL_WIDTH":"200.0","NODE_NESTED_NETWORK_IMAGE_VISIBLE":"true","NODE_PAINT":"#1E90FF","NODE_SELECTED":"false","NODE_SELECTED_PAINT":"#FFFF00","NODE_SHAPE":"ROUND_RECTANGLE","NODE_SIZE":"35.0","NODE_TRANSPARENCY":"255","NODE_VISIBLE":"true","NODE_WIDTH":"75.0","NODE_X_LOCATION":"0.0","NODE_Y_LOCATION":"0.0","NODE_Z_LOCATION":"0.0"},"dependencies":{"nodeCustomGraphicsSizeSync":"true","nodeSizeLocked":"false"},"mappings":{"NODE_LABEL":{"type":"PASSTHROUGH","definition":"COL=name,T=string"}}},{"properties_of":"edges:default","properties":{"EDGE_CURVED":"true","EDGE_LABEL_COLOR":"#000000","EDGE_LABEL_FONT_FACE":"Dialog.plain,plain,10","EDGE_LABEL_FONT_SIZE":"10","EDGE_LABEL_ROTATION":"0.0","EDGE_LABEL_TRANSPARENCY":"255","EDGE_LABEL_WIDTH":"200.0","EDGE_LINE_TYPE":"SOLID","EDGE_PAINT":"#323232","EDGE_SELECTED":"false","EDGE_SELECTED_PAINT":"#FF0000","EDGE_SOURCE_ARROW_SELECTED_PAINT":"#FFFF00","EDGE_SOURCE_ARROW_SHAPE":"NONE","EDGE_SOURCE_ARROW_SIZE":"6.0","EDGE_SOURCE_ARROW_UNSELECTED_PAINT":"#000000","EDGE_STACKING":"AUTO_BEND","EDGE_STACKING_DENSITY":"0.5","EDGE_STROKE_SELECTED_PAINT":"#FF0000","EDGE_STROKE_UNSELECTED_PAINT":"#848484","EDGE_TARGET_ARROW_SELECTED_PAINT":"#FFFF00","EDGE_TARGET_ARROW_SHAPE":"NONE","EDGE_TARGET_ARROW_SIZE":"6.0","EDGE_TARGET_ARROW_UNSELECTED_PAINT":"#000000","EDGE_TRANSPARENCY":"255","EDGE_UNSELECTED_PAINT":"#404040","EDGE_VISIBLE":"true","EDGE_WIDTH":"2.0","EDGE_Z_ORDER":"0.0"},"dependencies":{"arrowColorMatchesEdge":"false"}}]},{"cyHiddenAttributes":[{"n":"layoutAlgorithm","v":"yFiles Hierarchic Layout"}]},{"metaData":[{"name":"nodeAttributes","elementCount":8,"version":"1.0"},{"name":"cyGroups","version":"1.0"},{"name":"cyTableColumn","elementCount":15,"version":"1.0"},{"name":"edges","elementCount":10,"idCounter":212,"version":"1.0"},{"name":"cyVisualProperties","elementCount":3,"version":"1.0"},{"name":"cyHiddenAttributes","elementCount":1,"version":"1.0"},{"name":"nodes","elementCount":8,"idCounter":176,"version":"1.0"},{"name":"cyNetworkRelations","version":"1.0"},{"name":"networkAttributes","elementCount":1,"version":"1.0"},{"name":"cartesianLayout","elementCount":8,"version":"1.0"},{"name":"edgeAttributes","elementCount":60,"version":"1.0"},{"name":"tableVisualProperties","version":"1.0"},{"name":"cySubNetworks","version":"1.0"}]},{"status":[{"error":"","success":true}]}]