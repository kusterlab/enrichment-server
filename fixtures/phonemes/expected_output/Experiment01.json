{
  "pathway": {
    "name": "Experiment01"
  },
  "nodes": [
    {
      "id": 161,
      "geneNames": [
        "EGFR"
      ],
      "type": "gene_protein",
      "x": 7.03125,
      "y": -213.125
    },
    {
      "id": 164,
      "geneNames": [
        "MAP2K1"
      ],
      "type": "gene_protein",
      "x": 7.03125,
      "y": -148.125
    },
    {
      "id": 167,
      "geneNames": [
        "MAPK1"
      ],
      "type": "gene_protein",
      "x": 7.03125,
      "y": 156.875
    },
    {
      "id": 170,
      "geneNames": [
        "MAPK14"
      ],
      "type": "gene_protein",
      "x": -11.71875,
      "y": -68.125
    },
    {
      "id": 173,
      "geneNames": [
        "MAPKAPK2"
      ],
      "type": "gene_protein",
      "x": -45.46875,
      "y": 251.875
    },
    {
      "id": 176,
      "geneNames": [
        "MAPKAPK5"
      ],
      "type": "gene_protein",
      "x": 59.53125,
      "y": 251.875
    },
    {
      "id": 151,
      "geneNames": [
        "AKT2"
      ],
      "type": "gene_protein",
      "x": -11.71875,
      "y": 11.875
    },
    {
      "id": 158,
      "geneNames": [
        "MAP3K8"
      ],
      "type": "gene_protein",
      "x": -11.71875,
      "y": 76.875
    }
  ],
  "links": [
    {
      "id": "relation-212",
      "sourceId": 170,
      "targetId": 176,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-209",
      "sourceId": 170,
      "targetId": 173,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-206",
      "sourceId": 170,
      "targetId": 151,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-203",
      "sourceId": 167,
      "targetId": 176,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-200",
      "sourceId": 167,
      "targetId": 173,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-194",
      "sourceId": 164,
      "targetId": 170,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-191",
      "sourceId": 164,
      "targetId": 167,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-188",
      "sourceId": 161,
      "targetId": 164,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-197",
      "sourceId": 158,
      "targetId": 167,
      "types": [
        "other"
      ]
    },
    {
      "id": "relation-179",
      "sourceId": 151,
      "targetId": 158,
      "types": [
        "other"
      ]
    }
  ]
}